orte/mca/plm/rsh/plm_rsh_module.c

```diff
@@ -177,14 +178,59 @@ static void process_launch_list(int fd, short args, void *cbdata);
 static int num_in_progress=0;
 static opal_list_t launch_list;
 static opal_event_t launch_event;
+static char *rsh_agent_path=NULL;
+static char **rsh_agent_argv=NULL;
 
 /**
  * Init the module
  */
 static int rsh_init(void)
 {
+    char *tmp;
     int rc;
 
+    /* we were selected, so setup the launch agent */
+    if (mca_plm_rsh_component.using_qrsh) {
+        /* perform base setup for qrsh */
+        asprintf(&tmp, "%s/bin/%s", getenv("SGE_ROOT"), getenv("ARC"));
```

Comment:
Disregard -- I see there's NULL checks for these `getenv()`s in `_component.c`; i.e., `.using_qrsh` won't be true unless they're non-NULL.